src/utils.js

```diff
@@ -53,7 +52,7 @@ export function deepPreserveProps(instanceProps, componentProps) {
     popperOptions: {
       ...instanceProps.popperOptions,
       ...componentProps.popperOptions,
-      modifiers: uniqueByName([
```

Comment:
I think a couple of tests could help to make sure the deduplication works as intended